### ISSUE TITLE
Error when minifying dynamic import

### ISSUE THREAD

**Issue by sansSpoon** [Comandeer/rollup-plugin-babel-minify#137]
## Steps to reproduce the issue

1. run rollup build without rollup-plugin-babel-minify, build successful
2. add minify and get the error

```
src/index.js → public/module...
[!] (babel-minify plugin) SyntaxError: Error transforming bundle with 'babel-minify' plugin: unknown: Support for the experimental syntax 'dynamicImport' isn't currently enabled (94:7):

  92 |     key: "handleClick",
  93 |     value: function handleClick() {
> 94 |       import("./moduleA.js").then(function (module) {
     |       ^
  95 |         console.log(module.default);
  96 |       }).catch(function (err) {// Handle failure
  97 |       });

Add @babel/plugin-syntax-dynamic-import (https://git.io/vb4Sv) to the 'plugins' section of your Babel config to enable parsing.
undefined (94:6)
SyntaxError: Error transforming bundle with 'babel-minify' plugin: unknown: Support for the experimental syntax 'dynamicImport' isn't currently enabled (94:7):
```
Babelrc is:
```
{
	"presets": [
		["@babel/env", {
			"modules": false
		}],
		["@babel/preset-react"]
	],
	"plugins": [
		["@babel/plugin-syntax-dynamic-import"]
	]
}
```

Running  Babel via CLI `babel src/App.js -o compiled.js` with the following babelrc is successful:

```
{
	"presets": [
		["@babel/env", {
			"modules": false
		}],
		["@babel/preset-react"],
		["minify", {
			"mangle": {
				"topLevel": true,
			}
		}]
	],
	"plugins": [
		["@babel/plugin-syntax-dynamic-import"]
	]
}
```

Version of package: 6.1.1
Node.js & npm versions: 11.0.0, 6.4.1
Operating system: macOS 10.13.6

**Comment by Comandeer** [comment]
It's caused by the fact that ATM `rollup-plugin-babel-minify` does not support adding additional plugins to Babel. I'm going to add such ability (see #138), which should fix also this error.

Please note also that ATM `.babelrc` file is not supported and all configuration has to be passed as configuration option to the plugin itself.

**Comment by Comandeer** [comment]
I just released 6.2.0, which should fix the issue. Now you can pass needed plugin as an option to the `rollup-plugin-babel-minify`. Example `rollup.config.js`:

```javascript
import minify from 'rollup-plugin-babel-minify';

export default {
	input: 'index.js',
	plugins: [
		minify( {
			plugins: [
				'@babel/plugin-syntax-dynamic-import'
			]
		} )
	],
	output: {
		sourcemap: true,
		file: 'bundle.js',
		format: 'es'
	}
};
```
